### ISSUE TITLE
 iFrame has no accessible Name

### ISSUE THREAD

**Issue by jill-iterators** [CityOfBoston/Iterators#144]
Screengrab/videos (insert file names here)
2020 08 24, iFrame has no accessible Name.mp4

Link
https://d8-stg.boston.gov/departments/digital-team/test-page

Node # (if known)
17356

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > Mac 10.15.6 > Chrome 84

Trusted Tester Test
12) Page Title, Frames, iFrames

Brief Description
An iFrame on the Test page does not have any accessible name and title

Steps to reproduce
1)	Go to the test link page. Move downward to the section named ‘iFrame’
2)	00:00 Check the iFrame out for accessibility issues

Desired Result
•	The iFrame does not have any accessibility issues

Actual Result
•	The iFrame does not have any accessible name or title

**Comment by subaha-cob** [comment]
@jill-iterators how did you get to that page. That is a test page for developers whenever they are adding a new component and want to test it out. It is not a true page. @andrebtlima we should probably add a nofollow tag for this page.

**Comment by andrebtlima** [comment]
@subaha-cob presumably the issue here is that all iFrame components on the site will lack an accessible name or title. So I think I'll try to to test this by finding another iFrame component on another page and follow up in the comments of this ticket if it has the same issue.

**Comment by andrebtlima** [comment]
@subaha-cob here's an example other than the digital test page:

https://www.boston.gov/departments/parks-and-recreation/design-and-construction-boston-parks

**Comment by subaha-cob** [comment]
@andrebtlima I had to roll this one back. But if you check UAT it is there with my fix for auto height.

**Comment by subaha-cob** [comment]
@jill-iterators this is on the live site. So please test there unless I say otherwise. thanks.

**Comment by ollie-iterators** [comment]
This appears to be fixed
